gnomad/gnomadv4_cost_run_extract_rows.py

```diff
@@ -0,0 +1,37 @@
+# LT 1/09/2022
+
+## Cost estimation run on ~5% of partitions
+## annotate gnomADv4 with frequencies and variant QC standard fields
+## export the rows as a hail.Table on a cpg bucket
+
+import hail as hl
+from cpg_utils.hail_batch import output_path
+
+# connect to hail, requester_pays parameters are setup in the dataproc parameters
+hl.init(default_reference='GRCh38')
+
+# gnomAD v4 exomes VDS
+big_vds = hl.vds.read_vds('gs://gnomad/v4.0/raw/exomes/gnomad_v4.0.vds')
+
+# subset to ~5% of partitions (big_vds has 47960 partitions)
+small_vds = hl.vds.VariantDataset(
```

Comment:
would need proper checking that the resulting number of partitions is an integer, and since this piece of code is intended to be run once once, it seems superfluous: I would rather  hard code the number of partitions